move_base:
  local_costmap:
    global_frame: odom
    robot_base_frame: base_footprint
    update_frequency: 5.0
    publish_frequency: 2.0
    static_map: false
    rolling_window: true
    width: 6.0
    height: 6.0
    resolution: 0.05
    origin_x: 0.0
    origin_y: 0.0
    
    plugins:
      - {name: obstacle_layer, type: 'costmap_2d/ObstacleLayer'}
      - {name: inflation_layer, type: 'costmap_2d/InflationLayer'}
    
    obstacle_layer:
      observation_sources: laser_scan
      laser_scan: {sensor_frame: laser_link, data_type: LaserScan, topic: /limo/scan, marking: true, clearing: true}
      footprint_clearing_enabled: true
      max_obstacle_height: 2.0
      track_unknown_space: true
    
    inflation_layer:
      inflation_radius: 0.55
      cost_scaling_factor: 10.0